composer.json

```diff
@@ -9,7 +9,8 @@
 
         "league/tactician": "^1.0",
         "ramsey/uuid": "^3.0",
-        "coduo/php-to-string": "^2.0"
+        "coduo/php-to-string": "^2.0",
+        "doctrine/dbal": "^2.5"
```

Comment:
I think this should go to the dev dependencies, in other case we force everyone to install doctrine dbal, even if they are not going to use it